cpp/include/proton/terminus.hpp

```diff
@@ -99,14 +102,15 @@ class terminus {
 
     /// Obtain a reference to the AMQP dynamic node properties for the
     /// terminus.  See also lifetime_policy.
+    /// @deprecated in favor of dynamic_properties()??
     PN_CPP_EXTERN value node_properties() const;
 
-    /// Extension capabilities that are supported/requested
+    /// **Unsettled API** Extension capabilities that are supported/requested
     PN_CPP_EXTERN std::vector<symbol> capabilities() const;
 
-    /// Obtain the AMQP dynamic node properties for the
+    /// **Unsettled API** Obtain the AMQP dynamic node properties for the
     /// terminus as a standard map.
```

Comment:
@astitcher The documentation still says "standard map".

Reply:
Good catch